pkg/handler/processor/guesser/type_ite6.go

```diff
@@ -35,6 +35,8 @@ func (_ *ite6TypeGuesser) GuessDocumentType(blob []byte, format processor.Format
 				return processor.DocumentITE6Generic
 			} else if strings.HasPrefix(statement.PredicateType, "https://in-toto.io/attestation/certify/v0.1") {
 				return processor.DocumentITE6Generic
+			} else if strings.HasPrefix(statement.PredicateType, "http://in-toto.io/attestation/vuln/v0.1") {
```

Comment:
Do we want to be consistent with http vs. https? I don't think it really matters, but not sure.